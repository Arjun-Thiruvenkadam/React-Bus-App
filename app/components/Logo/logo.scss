.Logo {
  height: 80%;
  display: flex;
  padding: 0px 25px;
}

.Logo img {
  height: 100%;
  cursor: pointer;
}

.Logo p {
  margin: auto;
  padding-left: 5px;
  color: #0f3460;
  font-family: 'Sacramento', cursive;
  font-size: x-large;
  font-weight: bolder;
  cursor: pointer;
}
