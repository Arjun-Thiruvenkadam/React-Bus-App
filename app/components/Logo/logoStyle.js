/**
 *
 * LogoStyle
 *
 */

import styled from 'styled-components';

const LogoStyle = styled.div`
  height: 80%;
  display: flex;
  padding: 0px 25px;

  img {
    height: 100%;
    cursor: pointer;
  }

  p {
    margin: auto;
    padding-left: 5px;
    color: #0f3460;
    font-family: 'Sacramento', cursive;
    font-size: x-large;
    font-weight: bolder;
    cursor: pointer;
  }
`;

export default LogoStyle;
