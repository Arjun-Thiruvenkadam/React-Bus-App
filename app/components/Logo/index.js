/**
 *
 * Logo
 *
 */

import React from 'react';
import { FormattedMessage } from 'react-intl';
import messages from './messages';
import Bus from '../../images/bus.png';
import './logo.scss';

function Logo() {
  return (
    <div className="Logo">
      <img src={Bus} alt="Bus" />
      <p>
        <FormattedMessage {...messages.logo} />
      </p>
    </div>
  );
}

export default Logo;
