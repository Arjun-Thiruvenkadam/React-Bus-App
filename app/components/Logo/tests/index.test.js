import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';
import Logo from '../index';

describe('<Logo />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <Logo />
      </IntlProvider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
