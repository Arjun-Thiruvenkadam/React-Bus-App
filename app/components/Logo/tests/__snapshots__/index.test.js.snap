// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render and match the snapshot 1`] = `
<div
  class="Logo"
>
  <img
    alt="Bus"
    src="IMAGE_MOCK"
  />
  <p>
    <span>
      Aju's Bus
    </span>
  </p>
</div>
`;
